Introduction.md

```diff
@@ -2,60 +2,76 @@
-  * Peer review of each contribution by core experts improves quality and boosts developer skills
-  * Make company visible as an attractive employer
-  * Improve company's reputation, and with it the standing of developers in their communities
- * Get closer to the customer
-    * Establish new ways of collaborating with suppliers and customers
+### Build software faster and better
 
+Consuming open source software typically increases the development speed and decreases development costs since one builds upon existing code and a working and tested functionality. One risk however is that required features or bug fixes are not provided by the community as quickly as needed. To mitigate that risk, it might make sense to build up the required skills and create these bug fixes and/or features yourself. Contributing them back to the upstream projects has important benefits:
+
+* Integrating "own" features into upstream projects makes maintenance a lot easier
+* Upstream versions can be directly used in own products and services
+* More features are obtained in shorter period of time
+* Higher quality is achieved in shorter period of time
+* Support available from core experts
+
+### Exercise strategic influence
+
+Despite the benefits of open source software wrt. development velocity and quality mentioned above, contributing to open source projects can also be important from a strategic point of view. In open source world, reputation and the ability to influence is typically build up by engaging in the community and by contributing. Thus, contributions to open source projects can help to ...
+
+* influence the direction of upstream open source projects
+* gain (co)copyright on open source software packages
+* access to the creativity of everyone interested in software
+
+Open source communities (particulary those run by the big open source foundations) provide a neutral place for collaboration between companies and other organizations. Thus, an open source approach could offer new ways of collaboration with suppliers, customers, partner and even competitors, just to mention industry- or domain-specific projects such as [Linux Foundation Energy](https://www.lfenergy.org/) or [Eclipse Tractus-X](https://projects.eclipse.org/proposals/eclipse-tractus-x). Establishing open source communities can also be a powerful means to create and maintain ecosystems and to establish defacto standards.
+
+### Attract, grow and retain talents
+
+Software (and therefore also open source software) becomes more and more ubiqitous in many products and areas. Thus, for many companies (we are not only referring to software or cloud companies, but also to companies from other segments, such as traditional hardware producers who integrate software into their products more and more) it is crucial to have a skilled and motivated software development workforce. An open source strategy including open source contributions and community engagements support this:
```

Comment:
I would take this even wider and include all companies who are becoming dependent on software:

```suggestion
Software (and therefore also open source software) becomes more and more ubiqitous in many products and areas. Thus, for many companies (we are not only referring to software or cloud companies, but also to companies from other segments, such as traditional hardware producers who integrate software into their products more and more, or any other company which is becoming more dependent on software due to accelerating digital transformation) it is crucial to have a skilled and motivated software development workforce. An open source strategy including open source contributions and community engagements support this:
```